utils/emitter.ts

```diff
@@ -4,9 +4,11 @@ export enum UserInteractionEvents {
   CLOSE_MENUS = "closeMenus",
   SCROLL_TO_WHERE_TO_BUY = "scrollToWhereToBuy",
   SCROLL_TO_ROADMAP = "scrollToRoadmap",
+  SCROLL_TO_HARDFORK = "scrollToRoadmap",
```

Comment:
I think you have a double "scrollToRoadmap" here
```suggestion
  SCROLL_TO_HARDFORK = "scrollToHardfork",
```